misk-hibernate/src/test/kotlin/misk/hibernate/TransacterTest.kt

```diff
@@ -364,8 +364,12 @@ abstract class TransacterTest {
   }
 
   @Test
-  @Disabled("Uniqueness constraints aren't reliably enforced on Vitess")
   fun constraintViolationCausesTransactionToRollback() {
+    // Uniqueness constraints aren't reliably enforced on Vitess
+    if (transacter.config().type.isVitess) {
+      return
```

Comment:
If you do assumeTrue you'll get output that shows the test was skipped!

Reply:
ohhh nice, I've been looking for that!